ci/update_model_info_deparate.py

```diff
@@ -84,10 +84,11 @@ def update_model_info(
     model_info = get_json_dict(model_info_path)
 
     if bundle_name_with_version not in model_info.keys():
-        model_info[bundle_name_with_version] = {"checksum": "", "source": ""}
+        model_info[bundle_name_with_version] = {"checksum": "", "source": "", "description": ""}
```

Comment:
Hi @RomanFenioux , this script is an old version and is not used, can you revert the changes? Thanks